addons/languages/english/pack/src/main/res/xml/english_autotext.xml

```diff
@@ -50,6 +50,7 @@
     <word src="becauseyou">because you</word>
     <word src="becuase">because</word>
     <word src="becuse">because</word>
+    <word src="behxviour">behaviour</word>
```

Comment:
why is that a common typo?

Reply:
i made this typo myself and i noted it down. Looking it closely the h and the a are not that close to each other.  I can definately remove it.

Reply:
yes, please remove.

Reply:
done